### PR TITLE
added popup

### PR DESCRIPTION
**Pull request by AzizRAA**
popup window added to mobile and desktop versions
pops up when clicking on the see project button on the
Created dynamically with javascript index.js
_`ADDED VALIDATION AS WELL`_

### CODE REVIEW

**Comment by AzizRAA** [comment]
Hello, I'm sorry I left a message for you before sending this project for review because there's only one linter error. It's a Web hint, I already reached out to Student Succes and they recommend ignoring it since it's not relevant for this project, kindly check oject and the details of the error again, please
image
image

File: style.css [hunk 1]
```diff
@@ -842,3 +850,546 @@ nav ul li {
...
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%) scale(0);
+    transition: 200ms ease-in-out;
+    z-index: 10;
+    background-color: white;
+    width: 90%;
+    max-width: 90%;
+    height: 85%;
```
Comment (omarsalem7) [inline style.css:1171]: - [ ] [OPTIONAL] It will be better to change your height to `height: 75vh;` ,so you modal appear in all screen.

**Review by omarsalem7 — APPROVED** [review]
Hi @AzizRAA 😊, 

# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
- Descriptive pull request ✔️
- You project meet all the requirements ✔️

Cheers and Happy coding!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cwaku** [comment]
Hi @AzizRAA 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

This PR is about the popup window but you submitted it for form validation. Kindly use the right form in the dashboard to resubmit this PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AzizRAA** [comment]
> Hi @AzizRAA
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> This PR is about the popup window but you submitted it for form validation. Kindly use the right form in the dashboard to resubmit this PR
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Actually it is not invalid as I do not have much time left I did not created new branch for validation project

File: style.css (hunk 1 above)
Comment (AzizRAA) [reply style.css:1171]: Thanks, I will fix that.

**Review by omarsalem7 — APPROVED** [review]
Hi @AzizRAA 😊, 

# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

## Optional suggestions

- [ ] [OPTIONAL] For the next time, please use a branch for every pull request not the same one 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**